examples/browser-only/prepare-plugins.js

```diff
@@ -0,0 +1,42 @@
+const { promisify } = require('util');
+const glob = promisify(require('glob'));
+const fs = require('fs').promises;
+const path = require('path');
+
+async function run() {
+    // Resolve the `package.json` at the current working directory.
+    const pck = JSON.parse(await fs.readFile(path.resolve('package.json'), 'utf8'));
+
+    // Resolve the directory for which to download the plugins.
+    const pluginsDir = pck.theiaPluginsDir || 'extension';
+
+    // Find all `plugin/extension/*` directories.
+    const plugins = await glob(`${pluginsDir}/*/extension`);
+
+    for (const pluginExtensionPath of plugins) {
+        // Extract the plugin name from the parent folder of the extension.
+        const pluginName = path.basename(path.dirname(pluginExtensionPath)).replace(/[.\-]/g, '_');
+        const targetDir = path.join('lib', 'frontend', 'hostedPlugin', pluginName);
+
+        // Ensure the target directory exists.
+        await fs.mkdir(targetDir, { recursive: true });
```

Comment:
If the target directory already exists we should skip copying, or at least only copy missing files.